packages/instrumentation-exception/src/browser/instrumentations/HyperDXErrorInstrumentation.ts

```diff
@@ -136,7 +136,7 @@ export class HyperDXErrorInstrumentation extends InstrumentationBase {
     recordException(ev, {}, this.tracer, span);
   }
 
-  public report(
+  protected hdxReport(
```

Comment:
why was this public? InstrumentationBase doesn't define a `report` method 🤔

Reply:
not sure, very weird. nothing from the instrumentation should be public

Reply:
its used in otel-web (https://github.com/hyperdxio/hyperdx-js/blob/105caf78d52e1f6aec3968710947f2e6da02cf22/packages/otel-web/src/index.ts#L726). Its nice now we can  see it since they are all in the same repo